webapp/src/components/ParcelCoords/ParcelCoords.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { parcelType } from 'components/types'
+import Collapsable from 'components/Collapsable'
+import ParcelCoord from './ParcelCoord'
+import './ParcelCoords.css'
+
+const MAX_PARCEL_COORD_HEIGHT = 47
+
+export default class ParcelCoords extends React.PureComponent {
+  static propTypes = {
+    ...ParcelCoord.propTypes,
+    children: PropTypes.node,
+    parcels: PropTypes.arrayOf(parcelType),
+    isCollapsable: PropTypes.bool
+  }
+
+  static defaultProps = {
+    isCollapsable: true
+  }
+
+  render() {
+    const { children, parcels, isCollapsable, ...parcelCoordProps } = this.props
+    const maxHeight = isCollapsable ? MAX_PARCEL_COORD_HEIGHT : Infinity
```

Comment:
As `Collapsable` receive the `maxHeight` can we just send it as `null` if `isCollapsable = false` and remove `maxHeight: PropTypes.number.isRequired,` from Collapsable component. 

It will display `maxHeight: 100%` if not valid height is passed. Receive it as `Infinity` makes me go again to check what happen with a big height is passed

Reply:
You're right, it's not clear what the `Infinity` is doing here. So instead of having the `Collapsable` receive a `null` I just check on the father component if I have to have a `Collapsable`.

So instead of `isCollapsable ? MAX_PARCEL_COORD_HEIGHT : Infinity` we'd do
`return isCollapsable ? <Collapsable>div</Collapsable> : div`
which also saves us components and logic